evadb/third_party/databases/postgres/postgres_handler.py

```diff
@@ -0,0 +1,112 @@
+# coding=utf-8
+# Copyright 2018-2023 EvaDB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pandas as pd
+import psycopg2
```

Comment:
I dont think this will be installed by default in circleci since its under a separate header in setup.py

Reply:
Yes. We need to fix import psycopg2 across the code base. We should only import this if user is creating a postgres database connection.

Reply:
Add dynamic install and import here https://github.com/georgia-tech-db/evadb/pull/937/commits/106896b912fad8308af6f68e08f52f9691043385.

I will work on another PR to allow testing everything in CI.

Reply:
Thanks!